imageroot/actions/create-module/20expandconfig

```diff
@@ -102,6 +112,7 @@ http {
         # Remove X-Powered-By, which is an information leak
         fastcgi_hide_header X-Powered-By;
 
+
```

Comment:
Remove this changed line for a cleaner patch

Reply:
done